<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>accueil</class>
 <widget class="QDialog" name="accueil">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1158</width>
    <height>791</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	background-image: url(:/img/ressources/2.png);
	

		

font-family:century gothic;
font-size:18px;
}
QTabWidget
{
 background:#333;
}


QFrame
{
 background:#333;
	background-color: rgb(107, 107, 107);
	background-color: rgb(188, 193, 101);
	background-color: rgb(191, 165, 165);
	background-color: rgb(89, 14, 54);
	
}


QPushButton
{
background:red;
	background-color: rgb(255, 107, 114);
border-raduis:60px;
}
QLabel
{
color:white;
}
QPushButton
{
color:white;
border-radius:15px;
}
QPushButton:hover
{
color:#333;
border-raduis:15px;
background:#49ebff;
}
QLineEdit
{
background:transparent;
border:none;
color:#ffffff;
color: rgb(62, 124, 61);
	color: rgb(0, 255, 213);
	
border-button:3ptx solid #ffffff;
	color: rgb(6, 255, 56);
	
}
QComboBox
{
background:transparent;
border:none;
color:#717072;
border-bottom:1px solid #717072 }
QToolButton
{
background:red;
	background-color: rgb(255, 107, 114);
border-raduis:60px;
}
</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>721</width>
     <height>541</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(198, 126, 200);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>250</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Nom</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>320</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Mot De Passe</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>420</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Sign In</string>
    </property>
    <property name="icon">
     <iconset resource="ressources.qrc">
      <normaloff>:/img/ressources/login-symbol-of-an-arrow-entering-to-a-black-square_icon-icons.com_72992.png</normaloff>:/img/ressources/login-symbol-of-an-arrow-entering-to-a-black-square_icon-icons.com_72992.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>35</width>
      <height>35</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_nomlogin">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>250</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">	border-width: 1px; border-radius: 4px;
color: rgb(255, 255, 255);
	border-color: rgb(58, 58, 58);
	border-style: inset;
	padding: 0 8px;

	selection-background-color: rgb(187, 187, 187);
	selection-color: rgb(60, 63, 65);</string>
    </property>
    <property name="placeholderText">
     <string>nom</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_mdplogin">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>330</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">	border-width: 1px; border-radius: 4px;
color: rgb(255, 255, 255);
	border-color: rgb(58, 58, 58);
	border-style: inset;
	padding: 0 8px;

	selection-background-color: rgb(187, 187, 187);
	selection-color: rgb(60, 63, 65);
</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>mot de passe</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_voir">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>450</y>
      <width>361</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>191</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/img/ressources/logind.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>lineEdit_nomlogin</zorder>
   <zorder>lineEdit_mdplogin</zorder>
   <zorder>label_voir</zorder>
   <zorder>label_3</zorder>
   <zorder>pushButton_2</zorder>
  </widget>
 </widget>
 <resources>
  <include location="ressources.qrc"/>
 </resources>
 <connections/>
</ui>
